src/test/java/org/seqdoop/hadoop_bam/TestBAMInputFormat.java

```diff
@@ -26,11 +26,12 @@
   private TaskAttemptContext taskAttemptContext;
   private JobContext jobContext;
   
-  private void completeSetup(List<Interval> intervals) {
+  private void completeSetup(boolean boundedTraversal, List<Interval> intervals, boolean
```

Comment:
I know this is just test code, but with two boolean parameters it is not really clear in the test methods below which situation is under test.

How about adding new methods `setupBoundedTraversalUnplacedUnmapped(List<Interval>)` and similar that delegate to this one?